02/app.js

```diff
@@ -1,2 +1,16 @@
 const x = 5;
-let result = 0;
\ No newline at end of file
+let result = 0;
+let i = 0;
+
+for (let i=1; i<=5; i++) {
```

Comment:
Zwróć uwagę, że dwa razy deklarujesz tą samą zmienną.
Raczej będzie to mylące.

Proponowałbym użycie innej nazwy dla tzw. zmiennej sterującej lub wykorzystanie jej w obu przypadkach i ustawianie jej wartości w odpowiednim miejscu np.
```
let i;
for (i=1; i<=5; i++) { /* ... */ }
i = 0;
while(i<x) { /* ... */ }
```